src/reactivityHelpers.js

```diff
@@ -1,8 +1,7 @@
+import * as Vue from 'vue'
```

Comment:
**nitpick**: it's the only place where you've no used named imports strictly? Any reason why.

Reply:
Yes. I don't know whether there's a `set` export until I check on `version`, I can't import `set` conditionally, and I can't import `set` if it doesn't exist, so I do what is left, I import everything.